api/environments.md

```diff
@@ -8,19 +8,18 @@ Tradecloud has 3 environments available for customers:
 
 * [Production live environment](environments.md#production-environment)
 * [Acceptance test environment](environments.md#acceptance-test-environment)
-* [Stable environment](environments.md#stable-environment)
 
 ## Production environment
 
 There is one [production environment](https://api.tradecloud1.com/) a.k.a. LIVE environment. Use [https://api.tradecloud1.com/v2/api-connector/](https://api.tradecloud1.com/v2/api-connector/) to send order \(responses\).
 
 The latest features will be made available by the Tradecloud continuous delivery process.
 
-It has an availability SLA of 99,5% per month, measured 24/7, except for [planned production maintenance](environments.md#planned-maintenance)
+It has an availability SLA of 99,5% per month, measured 9:00 to 18:00 CE\(S\)T on working days.
```

Comment:
Note that as long as we don't define any consequences of not meeting this availability, this is in fact an SLO